api/app/GraphQL/Mutations/SubmitApplication.php

```diff
@@ -105,11 +106,16 @@ public function __invoke($_, array $args)
         }
 
         // all validation has successfully completed above, execute the core function of this resolver
+        // TODO - decide on a default expiry date, placeholder of year past submission
+        // add signature and submission, as well as update the set expiry date and status, update([]) not used due to not working correctly
```

Comment:
Interesting I was able to get it working with update. Not really a problem though.

Reply:
to clarify, it seemed to fail to update some fields and the `->save()` way fixed it

![image](https://user-images.githubusercontent.com/40485260/186456099-248ad2d3-8952-4b04-ad8b-be90b02ab85c.png)

resulted in 

![image](https://user-images.githubusercontent.com/40485260/186456197-f15cb9b6-ab1b-4d45-b5be-4de872607054.png)